Graphics/GraphicsEngineVulkan/CMakeLists.txt

```diff
@@ -245,7 +245,7 @@ elseif(PLATFORM_IOS)
     find_library(UI_KIT UIKit)
     list(APPEND PRIVATE_DEPENDENCIES ${CORE_GRAPHICS} ${METAL_LIBRARY} ${QUARTZ_CORE} ${UI_KIT})
 elseif(PLATFORM_LINUX)
-    find_library(Vulkan_LIBRARY NAMES vulkan HINTS "$ENV{VULKAN_SDK}/lib" "../../ThirdParty/vulkan/libs/linux")
```

Comment:
I tried this, and it is not this easy unfortunately. On Linux, symbol exporting rules are different, and this will cause linker errors when building tests. `--visibility hidden` does not help, and I did not investigate how to fix this.

Reply:
I already used dynamic loading for vulkan on linux.
Without VOLK linking may fail if mesh shader extension doesn't exist in vulkan.so